src/Env/Config/Variable/SingleValue.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Manalize project.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Manalize\Env\Config\Variable;
+
+/**
+ * A single-value variable.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+abstract class SingleValue implements Variable
+{
+    private $value;
+
+    /**
+     * @param string $value
+     */
+    public function __construct($value)
+    {
+        $this->value = $value;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getReplaces()
+    {
+        return [
+            $this->getPlaceholder() => $this->value,
+        ];
+    }
+
+    public static function createFromReplaces(array $replaces)
+    {
+        return new static($replaces[static::getPlaceholder()]);
+    }
+
+    /**
+     * @return string
+     */
+    abstract protected static function getPlaceholder();
```

Comment:
I don't know if we should really care, but according to http://php.net/manual/fr/migration52.incompatible.php : 

> Dropped abstract static class functions. Due to an oversight, PHP 5.0.x and 5.1.x allowed abstract static functions in classes. As of PHP 5.2.x, only interfaces can have them.

(I have the warning in PHPStorm). 
Maybe this should be moved in an interface ? :/